public/main.js

```diff
@@ -0,0 +1,68 @@
+const { app, BrowserWindow } = require('electron');
```

Comment:
Why does this PR include adding a new main.js?
I suspect that you may need to catch up to most recent version of `dev`